app/lib/decidim/overrides/verifications/authorizations_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Overrides
+    module Verifications
+      # This controller allows users to create and destroy their authorizations.
+      module AuthorizationsController
+        def create
+          enforce_permission_to :create, :authorization, authorization: authorization
+
+          @form = Decidim::AccessCodes::Verification::AuthorizationForm.from_params(
+            params.merge(user: current_user)
+          ).with_context(current_organization: current_organization)
+
+          Decidim::AccessCodes::Verification::ConfirmUserAuthorization.call(authorization, @form, session) do
+            on(:ok) do
+              user_as_private_space_participant
+              flash[:notice] = t("authorizations.create.success", scope: "decidim.access_codes.verification")
```

Comment:
The flash messages added in the `user_as_private_space_participant` method are not displayed because this line is overriding the variable. If we want to show them we should write something like:

```suggestion
              add_flash_notice_message(I18n.t("authorizations.create.success", scope: "decidim.access_codes.verification"))
              flash[:notice] = flash_notice_messages.join(" ")
```